docs/content/community/getting-help.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: Getting Help | Dagster
+description: Have questions about how to use Dagster? Hit a bug? Have a feature request? This page includes tips on what to do.
+---
+
+# Guide to Getting Help
+
+Have questions about how to use Dagster? Hit a bug? Have a feature request? The Dagster Labs team and the Dagster community make a best-effort attempt to respond, on a few different platforms.
+
+For Dagster Cloud users, an Enterprise contract also comes with a guaranteed SLA.
+
+### Searching for answers
+
+Search engines like Google generally do a good job at surfacing relevant Dagster docs, Github Discussions and Github Issues, so we recommend starting there.
+
+Unfortunately, ChatGPT is not currently a great resource for answering questions about Dagster, because the corpus it was trained on doesn’t include Dagster’s latest APIs.
```

Comment:
should we try link to @PedramNavid 's LLM tool? https://llm-support-bot-production.up.railway.app/

Reply:
I would say not yet, I am building out a Dagster pipeline to update it with docs + recent issues. Once that's done we can publish it?

Reply:
sounds good to me!